lib/src/emoji_cell.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:emoji_picker_flutter/emoji_picker_flutter.dart';
+import 'package:emoji_picker_flutter/src/triangle_decoration.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+/// A widget that represents an individual clickable emoji cell.
+/// Can have a long pressed listener [onSkinToneDialogRequested] that
+/// provides necessary data to show a skin tone popup.
+class EmojiCell extends StatelessWidget {
```

Comment:
I was thinking you want to extract this logic in order to use it outside of the emoji-keyboard but also inside. In this PR the EmojiCell is only used in your example. Does it make sense to use it internal as well?

Reply:
That was the plan. Somehow, when I was stashing code to move between my workstations, the main changes for the `default_emoji_picker_view.dart` file got lost. Luckily, I was able to recover it, so it's all good now.